From 4abc835e45743517a12f009bc5ef3f030eaee15a Mon Sep 17 00:00:00 2001
From: Even Xu <Feng.Xu@freescale.com>
Date: Tue, 27 Dec 2011 12:49:31 +0800
Subject: [PATCH 0798/2463] ENGR00170940 MX6Q: Add OV8820 MIPI as default
 module

OV8820 MIPI camera driver will be built as a module default.

Signed-off-by: Even Xu <b21019@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index e1c9573..afc2e26 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1371,6 +1371,7 @@ CONFIG_VIDEO_MXC_IPU_CAMERA=y
 CONFIG_MXC_CAMERA_OV3640=m
 CONFIG_MXC_CAMERA_OV5640=m
 CONFIG_MXC_CAMERA_OV5640_MIPI=m
+CONFIG_MXC_CAMERA_OV8820_MIPI=m
 CONFIG_MXC_CAMERA_OV5642=m
 CONFIG_MXC_IPU_PRP_VF_SDC=m
 CONFIG_MXC_IPU_PRP_ENC=m
-- 
1.7.7.4

